apps/vs-code-react/src/app/export/workflowsSelection/advancedOptions.tsx

```diff
@@ -42,6 +42,14 @@ export const AdvancedOptions: React.FC = () => {
       defaultMessage: 'Generate infrastructure templates',
       description: 'Generate infrastructure templates',
     }),
+    INTEGRATION_ACCOUNT_SOURCE: intl.formatMessage({
```

Comment:
@ecfan Could you please take a look at the wording for these strings? 
++ @wsilveiranz to be aware of it